test/sqllogictest/explain/decorrelated_plan_as_text.slt

```diff
@@ -26,131 +29,1035 @@ SELECT * FROM t WHERE a IS NOT NULL
+                      Where
+                        l1 =
+                          CrossJoin
+                            Get l0
+                            Get materialize.public.t
+  Where
+    l0 =
+      Constant
+        - ()
 
-%3 = Let l2 =
-| Get %2 (l1)
-| Distinct group=(#0)
+EOF
 
-%4 =
-| Get materialize.public.t (u1)
+# Test EXISTS subqueries.
+query T multiline
+EXPLAIN DECORRELATED PLAN AS TEXT FOR
+SELECT * FROM t WHERE EXISTS(SELECT * FROM mv WHERE t.a < mv.a) AND EXISTS(SELECT * FROM mv WHERE t.b > mv.b)
+----
+Let
+  Project #0, #1
+    Filter #2
+      Let
+        Project #0, #1, #3
+          Join
+            Get l4
+            Let
+              Union
+                Get l6
+                CrossJoin
+                  Project #0
+                    Join
+                      Union
+                        Negate
+                          Reduce group_by=[#0] monotonic=false
+                            Get l6
+                        Reduce group_by=[#0] monotonic=false
+                          Get l5
+                      Get l5
+                  Constant
+                    - (false)
+              Where
+                l6 =
+                  CrossJoin
+                    Reduce group_by=[#0] monotonic=false
+                      Filter (#0 > #2)
+                        CrossJoin
+                          Get l5
+                          Get materialize.public.mv
+                    Constant
+                      - (true)
+        Where
+          l5 =
+            Reduce group_by=[#1] monotonic=false
+              Get l4
+          l4 =
+            Project #0, #1
+              Filter #2
+                Let
+                  Project #0, #1, #3
+                    Join
+                      Get l1
+                      Let
+                        Union
+                          Get l3
+                          CrossJoin
+                            Project #0
+                              Join
+                                Union
+                                  Negate
+                                    Reduce group_by=[#0] monotonic=false
+                                      Get l3
+                                  Reduce group_by=[#0] monotonic=false
+                                    Get l2
+                                Get l2
+                            Constant
+                              - (false)
+                        Where
+                          l3 =
+                            CrossJoin
+                              Reduce group_by=[#0] monotonic=false
+                                Filter (#0 < #1)
+                                  CrossJoin
+                                    Get l2
+                                    Get materialize.public.mv
+                              Constant
+                                - (true)
+                  Where
+                    l2 =
+                      Reduce group_by=[#0] monotonic=false
```

Comment:
Well, it's because currently it is part of the `Reduce` :)

Now that we have the foundation for attribute derivation, I'm happy to follow up with that once we have the logic defined under `mz_transform::attribute`.